docs/source/high-level-arch.rst

```diff
@@ -0,0 +1,56 @@
+.. meta::
+   :description: TorchRec High Level Architecture
+   :keywords: recommendation systems, sharding, distributed training, torchrec, architecture
+
+==================
+TorchRec High Level Architecture
+==================
```

Comment:
can you please make sure that the underline is as long as the title - if it's shorter, it will add a warning "Underline is too short" to the build. You don't really need an overline can be ommitted.